rbd/mirror.go

```diff
@@ -710,10 +710,8 @@ func (iter *MirrorImageGlobalStatusIter) Next() (*GlobalMirrorImageIDAndStatus,
 }
 
 // Close terminates iteration regardless if iteration was completed and
-// frees any associated resources.
-func (iter *MirrorImageGlobalStatusIter) Close() error {
-	iter.buf = nil
-	iter.lastID = ""
+// frees any associated resources. (DEPRECATED)
+func (_ *MirrorImageGlobalStatusIter) Close() error {
```

Comment:
```suggestion
func (*MirrorImageGlobalStatusIter) Close() error {
```

should do the trick

Reply:
Hmm... interesting...

Reply:
BTW. I am using revive, but this seems to be not enabled by default. I only get it, when I use the config file.

Reply:
> I only get it, when I use the config file.

Oh ok. I pretty much always use it thru `make check` and that's how the CI calls it too. Fair warning: I'm probably going to enable even more checks in the near future too.